error_check.php

```diff
@@ -1,4 +1,62 @@
 <?php
+/**
+ * Checks if required fields are selected
+ *
+ * @param user_id The user requesting the course data
+ * @return String A sql command to get the requested data
+ */
+function get_courses($user_id){
+    //echo "SELECT courses.course_code, user_association.user_id, user_association.section_code FROM courses INNER JOIN user_association ON courses.course_code=user_association.course_id where user_id='$user_id'";
+    return "SELECT courses.course_code, user_association.user_id, user_association.section_code FROM courses INNER JOIN user_association ON courses.course_code=user_association.course_id where user_id='$user_id'";
+}
+
+// /**
+//  * Checks if required fields are selected
+//  *
+//  * @param user_id The user requesting the course data
+//  * @param survey_instance_id The survey request to view results
+//  * @return String A sql command to get the requested data
+//  */
+// function get_results($user_id, $survey_instance_id){
+//     //echo "SELECT courses.course_code, user_association.user_id, user_association.section_code FROM courses INNER JOIN user_association ON courses.course_code=user_association.course_id where user_id='$user_id'";
+//     return "SELECT questions.question_id, questions.answer_type, questions.content, GROUP_CONCAT(responses.answer SEPARATOR ', ') as answer from questions INNER JOIN responses ON responses.question_id=questions.question_id where survey_instance_id=$survey_instance_id;";
+//     //return "SELECT questions.question_id, questions.answer_type, questions.content, responses.answer from questions INNER JOIN responses ON responses.question_id=questions.question_id where survey_instance_id=$survey_instance_id;";
+// }
+
+/**
+ * Checks if required fields are selected
+ *
+ * @param user_id The user requesting the course data
+ * @param survey_instance_id The survey request to view results
+ * @return String A sql command to get the requested data
+ */
+function get_results(){
+    $user_id = $_GET["users"];
```

Comment:
Utility functions should avoid accessing global variables at all costs. If you need `$user_id` and `$ta_id`, pass that in when you call this function.